diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8e9c63b..239182f 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -145,6 +145,8 @@ find_package(hip REQUIRED PATHS /opt/rocm)
 target_flags(HIP_COMPILER_FLAGS hip::device)
 # Remove cuda arch flags
 string(REGEX REPLACE --cuda-gpu-arch=[a-z0-9]+ "" HIP_COMPILER_FLAGS "${HIP_COMPILER_FLAGS}")
+string(REGEX REPLACE --hip-device-lib-path=/usr/lib[^0-9]
+  "--hip-device-lib-path=/usr/lib64 " HIP_COMPILER_FLAGS "${HIP_COMPILER_FLAGS}")
 
 message(STATUS "Hip compiler flags: ${HIP_COMPILER_FLAGS}")
 
diff --git a/cmake/hip-config.cmake b/cmake/hip-config.cmake
index 279de67..903fbd6 100644
--- a/cmake/hip-config.cmake
+++ b/cmake/hip-config.cmake
@@ -85,11 +85,11 @@ set(GPU_TARGETS "${AMDGPU_TARGETS}" CACHE STRING "GPU targets to compile for")
 include( "${CMAKE_CURRENT_LIST_DIR}/hip-targets.cmake" )
 
 set_property(TARGET hip::device APPEND PROPERTY 
-  INTERFACE_COMPILE_OPTIONS -x hip --hip-device-lib-path=${AMD_DEVICE_LIBS_PREFIX}/lib
+  INTERFACE_COMPILE_OPTIONS -x hip --hip-device-lib-path=${AMD_DEVICE_LIBS_PREFIX}/lib64
 )
 
 set_property(TARGET hip::device APPEND PROPERTY 
-  INTERFACE_LINK_LIBRARIES --hip-device-lib-path=${AMD_DEVICE_LIBS_PREFIX}/lib --hip-link
+  INTERFACE_LINK_LIBRARIES --hip-device-lib-path=${AMD_DEVICE_LIBS_PREFIX}/lib64 --hip-link
 )
 
 set_property(TARGET hip::device APPEND PROPERTY 
