--- src/CMakeLists.txt	2020-11-27 18:28:45.608977180 -0500
+++ src/CMakeLists.txt	2020-11-29 22:43:54.116074936 -0500
@@ -4,7 +4,8 @@
 
 #cmake   -DOPENCL_DIR=/path to/opencl ..
 
-set(ROCCLR_EXPORTS_FILE "${CMAKE_CURRENT_BINARY_DIR}/amdrocclr_staticTargets.cmake")
+set(ROCCLR_EXPORTS_FILE
+  "${CMAKE_CURRENT_BINARY_DIR}/amdrocclr_staticTargets.cmake")
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake" "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 
@@ -202,7 +203,10 @@
 #endif()
 endif()
 
-install( TARGETS amdrocclr_static ARCHIVE DESTINATION lib LIBRARY DESTINATION lib COMPONENT applications)
+install( TARGETS amdrocclr_static EXPORT amdrocclr_static_export ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT applications )
+install( EXPORT amdrocclr_static_export FILE amdrocclr_staticTargets.cmake
+  DESTINATION ${CMAKE_INSTALL_LIBDIR})
 file(GLOB headerfiles "${CMAKE_CURRENT_SOURCE_DIR}/*.h*")
 install (
     FILES ${headerfiles}
