diff --git a/cmake/public/LoadHIP.cmake b/cmake/public/LoadHIP.cmake
index 34902db..f3c2399 100644
--- a/cmake/public/LoadHIP.cmake
+++ b/cmake/public/LoadHIP.cmake
@@ -190,7 +190,9 @@ if(HIP_FOUND)
   find_library(ROCM_HIPRTC_LIB hiprtc HINTS ${HIP_PATH}/lib)
   # roctx is part of roctracer
   find_library(ROCM_ROCTX_LIB roctx64 HINTS ${ROCTRACER_PATH}/lib)
-  set(roctracer_INCLUDE_DIRS ${ROCTRACER_PATH}/include)
+  set(roctracer_INCLUDE_DIRS ${CMAKE_INSTALL_PREFIX}/include/roctracer)
+  message(STATUS "ROCM_ROCTX_LIB is ${ROCM_ROCTX_LIB}")
+  message(STATUS "roctracer_INCLUDE_DIRS is ${roctracer_INCLUDE_DIRS}")
 
   # Necessary includes for building PyTorch since we include HIP headers that depend on hcc/hsa headers.
   set(hcc_INCLUDE_DIRS ${HCC_PATH}/include)
