--- src/CMakeLists.txt	2020-11-27 20:16:31.690698941 -0500
+++ src/CMakeLists.txt	2020-11-27 20:19:04.053692384 -0500
@@ -234,14 +234,15 @@
 # Generate profiling API macros/structures header
 if(HIP_PLATFORM STREQUAL "hcc")
 if(USE_PROF_API EQUAL 1)
-set(PROF_API_STR "${PROJECT_BINARY_DIR}/include/hip/hcc_detail/hip_prof_str.h")
+set(PROF_API_PATH "${PROJECT_BINARY_DIR}/include/hip/hcc_detail")
+set(PROF_API_STR "${PROF_API_PATH}/hip_prof_str.h")
 set(PROF_API_HDR "${CMAKE_CURRENT_SOURCE_DIR}/include/hip/hcc_detail/hip_runtime_api.h")
 set(PROF_API_SRC "${CMAKE_CURRENT_SOURCE_DIR}/src")
 set(PROF_API_GEN "${CMAKE_CURRENT_SOURCE_DIR}/hip_prof_gen.py")
 set(PROF_API_LOG "${PROJECT_BINARY_DIR}/hip_prof_gen.log.txt")
 set(PROF_API_CMD "${PROF_API_GEN} -v ${OPT_PROF_API} ${PROF_API_HDR} ${PROF_API_SRC} ${PROF_API_STR} >${PROF_API_LOG}")
 MESSAGE(STATUS "Generating profiling promitives: ${PROF_API_STR}")
-execute_process(COMMAND sh -c "rm -f ${PROF_API_STR}; ${PROF_API_CMD}")
+execute_process(COMMAND sh -c "rm -f ${PROF_API_STR}; mkdir -p ${PROF_API_PATH}; ${PROF_API_CMD}")
 set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${PROF_API_GEN} ${PROF_API_HDR} ${PROF_API_STR})
 
 # Enable profiling API
