--- src/rocclr/CMakeLists.txt	2020-11-29 20:42:19.927388809 -0500
+++ src/rocclr/CMakeLists.txt	2020-11-29 21:15:41.009302702 -0500
@@ -120,7 +120,7 @@
     )
     MESSAGE(STATUS "Code Object Manager found at ${amd_comgr_DIR}.")
 
-include_directories("$<TARGET_PROPERTY:amd_comgr,INTERFACE_INCLUDE_DIRECTORIES>")
+include_directories($<TARGET_PROPERTY:amd_comgr,INTERFACE_INCLUDE_DIRECTORIES>)
 
 add_definitions(-DBSD_LIBELF)
 
@@ -146,6 +146,9 @@
  fixme.cpp
  )
 set_target_properties(hip64 PROPERTIES POSITION_INDEPENDENT_CODE ON)
+target_include_directories(hip64 PRIVATE $<TARGET_PROPERTY:amdrocclr_static,INTERFACE_INCLUDE_DIRECTORIES>)
+target_compile_definitions(hip64 PRIVATE
+  $<TARGET_PROPERTY:amdrocclr_static,INTERFACE_COMPILE_DEFINITIONS>)
 
 set_target_properties(
     hip64 PROPERTIES
@@ -206,7 +209,8 @@
     amdhip64_static_combiner
     ALL
     COMMAND rm -f $<TARGET_FILE:amdhip64_static>  # Must remove old one, otherwise the new one will contain obsolete stuff
-    COMMAND ${CMAKE_AR} -rcsT $<TARGET_FILE:amdhip64_static> $<TARGET_FILE:amdhip64_static_temp> $<TARGET_FILE:amdrocclr_static>
+    COMMAND ${CMAKE_AR} -x $<TARGET_FILE:amdrocclr_static>
+    COMMAND ${CMAKE_AR} -rcsT $<TARGET_FILE:amdhip64_static> $<TARGET_FILE:amdhip64_static_temp>
     DEPENDS amdhip64_static amdhip64_static_temp amdrocclr_static # To make sure this is the last step
     COMMENT "Combining static libs into amdhip64_static"
 )
